documentation/src/main/asciidoc/topics/ref_rest_data_formats.adoc

```diff
@@ -4,10 +4,10 @@
 https://en.wikipedia.org/wiki/Media_type[MediaType].
 
 //Community Link
-ifndef::productized[]
+ifdef::community[]
```

Comment:
@gustavonalle I wonder if this ifdef is still necessary

Reply:
Well spotted @danberindei I've changed the link to `link:{dev_docs}#data_encoding[Encoding]` so it can be used downstream without the ifdef.